ui/packages/app/src/Blocks.svelte

```diff
@@ -35,6 +35,7 @@
 	export let target: HTMLElement;
 	export let id: number = 0;
 	export let autoscroll: boolean = false;
+	export let show_api: boolean = true;
```

Comment:
I think this is backwards compatible. I tested by manually removing `show_api` from the config and this parameter is still set to true. I think that covers us in the case you try to load a config from a demo created in a previous version that doesn't have `show_api`. 

Does this sound right to you @aliabid94 ?

Reply:
sounds good to me